src/main/java/com/cloudogu/smeagol/repository/domain/Repository.java

```diff
@@ -0,0 +1,63 @@
+package com.cloudogu.smeagol.repository.domain;
+
+import com.google.common.base.MoreObjects;
+
+import java.time.Instant;
+import java.util.Objects;
+
+/**
+ * Repository is an aggregate which represents a source code repository.
+ */
+public class Repository {
+
+    private RepositoryId id;
+    private Name name;
+    private Description description;
+    private Instant lastModified;
+
+    public Repository(RepositoryId id, Name name, Description description, Instant lastModified) {
+        this.id = id;
+        this.name = name;
+        this.description = description;
+        this.lastModified = lastModified;
+    }
+
+    public RepositoryId getId() {
+        return id;
+    }
+
+    public Name getName() {
+        return name;
+    }
+
+    public Description getDescription() {
+        return description;
+    }
+
+    public Instant getLastModified() {
+        return lastModified;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS00122)